stable/gocd/values.yaml

```diff
@@ -223,6 +223,13 @@ agent:
   #     cpu: 100m
   #     memory: 1024Mi
 
+  # agent.hostAliases allows the modification of the hosts file inside a container
+  hostAliases:
```

Comment:
Just a quick question because I haven't tried this out. What happens if the replicaCount is more than 1? Will all the agent containers that are brought up have the same host name?

Reply:
@varshavaradarajan: It's not the host name of the container, but what's inside the `/etc/hosts` file inside each container. 

Should be fine (and expected) to have that as the same in multiple containers.

Reply:
For more details, see: https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/